### ISSUE TITLE
Column headings

### ISSUE THREAD

**Issue by adz35** [cbx33/garzolean#4]
Is it possible to have the column headings interchangeable so they can say whatever we like? Be quite useful also if i wanted to display a different field on the data or additional fields. ie Customer/order#/Estimated Trolley count etc etc.
Some customers might require additional info so having the option will be very beneficial.

Thanks :)

**Comment by cbx33** [comment]
Yes totally possible. Probably a Thursday job. Will do my best to get to it then.
